src/rest/mormot.rest.mvc.pas

```diff
@@ -2054,6 +2065,17 @@ procedure TMvcRunOnRestServer.RunOnRestServerSub(Ctxt: TRestServerUriContext);
     InternalRunOnRestServer(Ctxt, Ctxt.UriBlobFieldName);
 end;
 
+procedure TMvcRunOnRestServer.UnregisterMethods;

```

Comment:
perhaps move it to TRestServer itself

Reply:
Does that make any sense? This code is for unregistering all methods introduced by the MVC interface. Moving to TRestServer would unregister ALL methods which is not what we want here. Imho this is right place.